README.md

```diff
@@ -1,2 +1,199 @@
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 About <a name="about-project"></a>
+
+A web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.
+
+## 🛠 Built With <a name="built-with"></a> 
+
+### Tech Stack <a name="tech-stack"></a>
+- React
+- Redux
+
+### Database <a name="data-base"></a>
+- API dependdencies
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#"></a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">N/A</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="#">N/A</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Feature <a name="key-features"></a>
+ - Book rockets
+ - Join selected space missions
+
+<p align="right">(<a href="#table-contents">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a> 
+
+[Space Traveler's Hub](https://space-travellers-ztro.onrender.com/)
+
+<p align="right">(<a href="#table-contents">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get the content of this project locally you need to run the following commands in your terminal.
+
+### Prerequisites
+
+- IDE (code editor) like: VScode, Sublime, etc
+- [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/)
+
+
+### Setup
+
+1. Download the **Zip** file or clone the repo with:
+```bash
+git clone https://github.com/maov19/space-travelers-hub/
+```
+2. To access cloned directory run:
+```bash
+cd space-travelers-hub
+```
+
+### Install
+
+> To install linters and other project's dependencies run:
+
+```bash
+npm install
+```
+
+### Usage
+
+To run the project, execute the following command: 
+
+- Excute below command in the terminal to serve live 
+```bash
+npm run dev
+```
+
+## Run Test for Linters
+
+For tracking linter errors locally you need to follow these steps:
+
+1. For tracking linter errors in HTML files run:
+```bash 
+npx hint .
+```
+
+2. For tracking linter errors in CSS or SASS files run:
+
+```bash
+npx stylelint "\*_/_.{css,scss}"
+```
+
+3. For tracking linter errors in JavaScript files run:
+
+```bash
+npx eslint .
+```
+
+### Deployment
+
+[Space Traveler's Hub](https://space-travellers-ztro.onrender.com/)
+
+<p align="right">(<a href="#table-contents">back to top</a>)</p>
+
+<!-- AUTHOR -->
+
+## 👥 Author: <a name="authors"></a>
+
+👤 **Authors**
+
+Koshao Kazungu Godfrey & Martin Ossa
+- GitHub: [@Koshao-sys](https://github.com/Koshao-sys)
+- Twitter: [@KoshaoKG](https://twitter.com/KoshaoKG)
+
+- GitHub: [@maov19](https://github.com/maov19)
+- Twitter: [@maov08](https://twitter.com/@maov08)
```

Comment:
I think that it'll be better to list out separately the authors as it appears on the [readme template](https://github.com/microverseinc/readme-template#-authors-).

Reply:
hi @emmiiorji ! thanks for the feedback.
Fixed!
![image](https://user-images.githubusercontent.com/108192399/231747039-7691cd88-c784-44ed-b051-6718d6eedca2.png)